NDTensors/ext/examples/NDTensorCUDA.jl

```diff
@@ -0,0 +1,128 @@
+using CUDA
+using NDTensors
+
+using ITensors
+using Test
+
+using Pkg;
+Pkg.add("Zygote");
```

Comment:
We shouldn't `add` packages like this in examples (users should install it themselves).